src/components/TeacherComponents/LessonOverviewVideo/LessonOverviewVideo.tsx

```diff
@@ -49,6 +49,7 @@ export const LessonOverviewVideo: FC<LessonOverviewVideoProps> = ({
           title={title}
           location={"lesson"}
           isLegacy={isLegacy}
+          defaultHiddenCaptions={signLanguageOn}
```

Comment:
Is there a linked requirement for this change in Notion? It doesn't seem related to the scope of the PR.